specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/bigDataPool.json

```diff
@@ -446,7 +446,12 @@
         "nodeCount": {
           "type": "integer",
           "format": "int32",
-          "description": "The number of nodes in the Big Data pool."
+          "description": "The number of nodes in the Big Data pool.",
+          "x-ms-mutability": [
```

Comment:
Can you please share the S360 link that shows up because of nodeCount property?

Reply:
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=18B5CFC3-39F6-4BF0-9725-8E9471DC8969

ROUNDTRIP_ADDITIONAL_PROPERTY

The property is returned in the GET response, but it is not declared in the PUT request. If it is a read-only property, update the swagger definition for this property to mark it as "readOnly": true. Alternatively, keep the property in the GET schema but remove it from the PUT schema. If it is immutable (its value cannot be changed after creation), update the swagger definition for this property to mark it with the "x-ms-mutability": ["create", "read"] annotation. If the property has a default value, update the Swagger definition for this property to mark it with "default": <default value> annotation.